src/org/traccar/reports/model/GeneralReport.java

```diff
@@ -0,0 +1,42 @@
+package org.traccar.reports.model;
+
+public class GeneralReport {
```

Comment:
Maybe rename it to `SummaryReport`? Because general sounds like some base class for all reports.